### ISSUE TITLE
Rest service - User has been authenticated but no Mapping found for HTTP request

### ISSUE THREAD

**Issue by arashkhalesi** [szerhusenBC/jwt-spring-security-demo#36]
I am doing rest modules in Micro-services No JSP or Login page. 
User is authenticated 
[JwtAuthenticationTokenFilter.java:56]authenticated user admin, setting security context

PageNotFound [DispatcherServlet.java:1172]No mapping found for HTTP request with URI [/rest-moudle/test] in DispatcherServlet with name 'dispatcherServlet'

Below is my WebConfig. configure(HttpSecurity httpSecurity)
I pretty much opened every incoming request no matter what it is but still does not find the mapping page.
.antMatchers(HttpMethod.GET, "/*").permitAll().antMatchers(HttpMethod.POST, "/*").permitAll()
.antMatchers(HttpMethod.GET, "/**").permitAll().antMatchers(HttpMethod.POST, "/**").permitAll()
.antMatchers(HttpMethod.GET, "/").permitAll().antMatchers(HttpMethod.POST, "/").permitAll()

**Comment by jmw5598** [comment]
@arashkhalesi 
Have you possibly mistyped the targeted endpoint? Did you mean to `/rest-module/test` and not `/rest-moudle/test`.

Also for your security config

     .antMatchers("/**").permitAll()

**Comment by arashkhalesi** [comment]
Thanks for quick response. I just typed that in my message as I did not want to post my actual url in public. 
I had this problem while ago in Spring 4  basic auth  while with no reason it did not work with URLs which was defined  as static in another class.
like @RequestMapping value=MyConstantURL.login) 
I am going to dig in to my code again to see maybe something is missing . however I appreciate your quick feedback :) 
I will let you know as soon as  fixed it   

    On Thursday, April 27, 2017 7:05 PM, Jason White <notifications@github.com> wrote:
 

 @arashkhalesi
Have you possibly mistyped the targeted endpoint? Did you mean to /rest-module/test and not /rest-moudle/test.Also for your security config .antMatchers("/**").permitAll()
